src/components/Content/TaskHierarchy/Task/Task.js

```diff
@@ -129,19 +157,22 @@ class RawTask extends Component {
     }
   }
 
+
   render() {
-    const { pursuances, taskData, match: { params: { pursuanceId } } } = this.props;
+    const { pursuances, taskData, autoComplete, currentPursuanceId } = this.props;
+    const { showChildren, showTaskForm, assignedTo, showAssigneeInput } = this.state;
```

Comment:
@Moe-Shoman The dev server says that `assignedTo` and `showTaskForm` are no longer used.  It makes sense that `assignedTo` shouldn't be used anymore -- we're using `assignedToName` instead -- but what about `showTaskForm`?

Reply:
Does it have something to do with top-level `TaskForm` being eliminated?

Reply:
I removed `showTaskForm` in ee7f5673b877883903fda71f52274230f9e6026e, but I'm still not sure what it did :-)

Reply:
It was used to toggle the form.
`{showTaskForm && <TaskForm parentGid={task.gid} />}`